Packs/ShadowxSOCAI/Integrations/ShadowxSOCAI/ShadowxSOCAI.yml

```
@@ -0,0 +1,211 @@
+commonfields:
+  id: ShadowxSOCAI
+  version: -1
+name: ShadowxSOCAI
+display: ShadowX SOCAI
+category: Analytics & SIEM
+description: ShadowX AI plugin that analyzes security logs and returns response, recommendation, risk/result, and prediction score. Supports API Key (JSON /Api/ endpoints) and cookie (HTML) modes.
+fromversion: 6.10.0
+
+configuration:
+- display: Server URL (e.g., https://app.shadowx.ai)
+  name: url
+  type: 0
+  required: true
+
+# API Key (Bearer) mode
+- display: API Key (recommended). If set, email/password are ignored.
+  name: api_key
+  type: 4
+  required: false
```

we no longer use type 4 for encrypted data, and please make sure to change retrieving the api key in the code file as well and replace it with, params.get("credentials", {}).get("password")
```suggestion
- displaypassword: API Key (recommended). If set, email/password are ignored.
  name: credentials
  type: 9
  hiddenusername: true
  required: false
```